avocado/core/messages.py

```diff
@@ -228,6 +228,9 @@ def handle(self, message, task, job):
         This assumes that the log message will not contain a newline, and thus
         one is explicitly added here.
         """
+        if task.metadata.get('logfile') is None:
+            task.metadata['logfile'] = os.path.join(task.metadata['task_path'],
+                                                    'debug.log')
```

Comment:
I do have a feeling that this could be part of a configuration option, but I'm with postponing this.